spec.html

```diff
@@ -32412,6 +32562,101 @@ <h1>Expanded Years</h1>
           </emu-note>
         </emu-clause>
       </emu-clause>
+
+      <emu-clause id="sec-time-zone-offset-strings">
+        <h1>Time Zone Offset String Format</h1>
+
+        <p>
+          ECMAScript defines a string interchange format for the UTC offset of a time zone.
+          The format is described by the following grammar:
+        </p>
+
+        <emu-grammar type="definition">
+          ASCIISign ::: one of
+            `+` `-`
+
+          Sign :::
+            ASCIISign
+            U+2212
+
+          Hour :::
+            `0` DecimalDigit
+            `1` DecimalDigit
+            `20`
+            `21`
+            `22`
+            `23`
+
+          MinuteSecond :::
+            `0` DecimalDigit
+            `1` DecimalDigit
+            `2` DecimalDigit
+            `3` DecimalDigit
+            `4` DecimalDigit
+            `5` DecimalDigit
+
+          DecimalSeparator ::: one of
+            `.` `,`
+
+          TimeSeparator[Extended] :::
+            [+Extended] `:`
+            [~Extended] [empty]
+
+          FractionalPart :::
+            DecimalDigit DecimalDigit? DecimalDigit? DecimalDigit? DecimalDigit? DecimalDigit? DecimalDigit? DecimalDigit? DecimalDigit?
+
+          Fraction :::
+            DecimalSeparator FractionalPart
+
+          UTCOffsetOptionalComponents[Extended] :::
+            TimeSeparator[?Extended] MinuteSecond
+            TimeSeparator[?Extended] MinuteSecond TimeSeparator[?Extended] MinuteSecond Fraction?
+
+          UTCOffsetString :::
```

Comment:
Suggested renames:
* |_UTCOffsetString_| to |_UTCOffset_|
* |_UTCOffsetOptionalComponents_| to something like |_HourSubcomponents_|
* |_Fraction_| to |_TemporalDecimalFraction_| (and subsume |_FractionalPart_|)
* |_DecimalSeparator_| to |_TemporalDecimalSeparator_|
* |_Sign_| to |_TemporalSign_|

Reply:
I've gone with this for now, but are you sure about having the "Temporal" name in there? It might be confusing since Temporal isn't part of the specification yet.

Reply:
Ahead of the Temporal proposal landing, "temporal" still makes sense as a generic adjective indicating relationship to time—necessary here because date and time representations can use `,` as a decimal separator and `–` as a negative sign, neither of which are true of non-temporal (i.e., non-time-related) numeric strings in ECMAScript.